packages/core/src/components/tabs/_tabs.scss

```
@@ -95,6 +95,13 @@ $tab-indicator-width: 3px !default;
   margin: 0;
   padding: 0;
   position: relative;
+
+  .#{$ns}-tabs:not(.#{$ns}-vertical) & > .#{$ns}-flex-expander {
```

Is the parent selector needed here?
```suggestion
  .#{$ns}-tabs:not(.#{$ns}-vertical) > .#{$ns}-flex-expander {
```

---

yea good catch I dunno what I was doing here - maybe originally had it nested with something else and removed the nesting without removing this

these should be equivalent right?